load_data.js

```diff
@@ -0,0 +1,74 @@
+(function(window){
+  window.extractData = function() {
+    var ret = $.Deferred();
+
+    FHIR.oauth2.ready(onReady, onError);
+
+    function onReady(smart){
+      var patient = smart.patient;

```

Comment:
If you get the chance, can you take a glance at the [style guide](https://github.com/airbnb/javascript) we have chosen to follow for JS? It'd be good to showcase SMART apps using the airbnb JS style guide. Some things I see we can maybe change include: 

- variable types (instead of `var`, using `const` or `let`)
- comma placement in your `defaultPatient` function
- formatting of brace/parenthesis extractData function call in the index.html file

They have linters you can setup to help catch some of these little style rules [here](https://github.com/airbnb/javascript#resources).